docs/monitor.asciidoc

```diff
@@ -1,23 +1,9 @@
-[[monitor-apm]]
-=== Monitor APM Server
-
-Use the {stack} {monitor-features} to gain insight into the real-time health and performance of APM Server.
-Stack monitoring exposes key metrics, like intake response count, intake error rate, output event rate,
-output failed event rate, and more.
-
-[float]
-[[monitor-apm-cloud]]
-=== Monitor APM running on {ecloud}
-
-{ecloud} manages the installation and configuration of a monitoring agent for you -- so
-all you have to do is flip a switch and watch the data pour in.
-
-* **{ess}** user? See {ece-ref}/ece-enable-logging-and-monitoring.html[ESS: Enable logging and monitoring].
-* **{ece}** user? See {cloud}/ec-enable-logging-and-monitoring.html[ECE: Enable logging and monitoring].
-
-[float]
 [[monitor-apm-self-install]]
-=== Monitor a self-installation of APM
+=== Monitor a Fleet-managed APM Server
+
+++++
+<titleabbrev>Fleet-managed</titleabbrev>
+++++
```

Comment:
> . Start {agent}
> +
> Use the command that work with your system:

The third step in this guide is actually not true anymore. Users are required to [re-enroll the elastic agent](https://www.elastic.co/guide/en/fleet/current/elastic-agent-cmd-options.html#elastic-agent-enroll-command) after changing the `elastic-agent.yml`, otherwise the changes won't have an effect.